bot.py

```diff
@@ -71,22 +71,26 @@ async def on_member_join(member):
     await channel.send(response)
 
 
-@bot.command(name="Welcome")
-async def welcome(ctx):
-    print(f"welcome heard")
-
-    # old/
-    # if message.author == client.user:
-    #    return
-    # if message.content == '!Welcome':
-    #   response = "Howdy"
-    # await message.channel.send(response)
-    # elif message.content == 'raise-exception':
-    # raise discord.DiscordException
-    # old/
-
-    response = f"Howdy {ctx.author}"
-    await ctx.send(response)
+@bot.command(name="roll")
+async def roll(ctx, die: str):
+    try:
+        num_of_dice, die_sides = die.split("d")
+        print(num_of_dice)

```

Comment:
are these print statements still needed?

Reply:
Yeah rn I'm using them to log to the console what the user tries to roll
I'm eventually gonna output that to a log file but for now, I'm print logging